src/Npgsql/NpgsqlFactory.cs

```diff
@@ -71,7 +71,7 @@ public sealed class NpgsqlFactory : DbProviderFactory, IServiceProvider
         /// <param name="serviceType">An object that specifies the type of service object to get.</param>
         /// <returns>A service object of type serviceType, or null if there is no service object of type serviceType.</returns>
 #nullable disable
-        public object GetService(Type serviceType)
+        public object? GetService(Type serviceType)
```

Comment:
Note that nullability is still disabled here (nullable disable), so the question mark should be generating a warning.

When null annotating I tried to avoid annotating leave public ADO.NET APIs such as these outside null contexts, because it would be good for them to be annotated in ADO.NET first (in case there's a conflict later). I do think it's OK to annotate for the obvious cases, and I also may have not been 100% consistent when annotating.

Reply:
`GetService` could return `null`, see https://docs.microsoft.com/en-us/dotnet/api/system.iserviceprovider.getservice?view=netstandard-2.1.

Reply:
Yes, I agree this should be nullable - but then you need to remove the `#nullable disable`